.repositories.yaml

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# List of repositories of Eclipse Che components
+---
+repositories:
+  - url: https://github.com/eclipse/che
+    name: che
+    description: >-
+      (this repository) the main project repository
+    devfile: https://github.com/eclipse/che/blob/master/devfile.yaml
+    doc: https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#che-server-aka-ws-master
+    components: 
+     - name: dockerfiles
+       url: https://github.com/eclipse/che/tree/master/dockerfiles
+       description: >-
+         source code, dockerfiles to build our main docker images. Note that Che-theia related dockerfiles are located in che-theia repo.
+     - name: orchestrator
```

Comment:
controller?

Reply:
or che-api

Reply:
I keep for now wsmaster, will track change of the name in another issue